scheduler/db/scheduler.changelog-master.yaml

```diff
@@ -134,6 +139,11 @@ databaseChangeLog:
                   type: integer
                   constraints:
                     nullable: true
+              - column:
+                  name: guid
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
```

Comment:
I guess this can be postponed... while at some point, we will need to have  a 'compact' before releasing out